use_cases/eluc/prescriptors/nsga2/nsga2_utils.py

```diff
@@ -78,11 +78,14 @@ def calculate_crowding_distance(front):
             else:
                 sorted_front[i].distance += 0
 
-def dominates(candidate1: Candidate, candidate2: Candidate):
+def dominates(candidate1: Candidate, candidate2: Candidate) -> bool:
     """
     Determine if one individual dominates another.
     """
+    better = False
     for obj1, obj2 in zip(candidate1.metrics, candidate2.metrics):
-        if obj1 > obj2:
+        if obj1 < obj2:
```

Comment:
Any unit test to validate that function? Does it depend on whether we maximize or minimize?

Reply:
In one case we return False, in the other we set `better` to True and return it later.
Should if be:
```
if obj1 <= obj2:
    return False
else:
    return True
```
Which is equivalent to:
```
return obj1 > obj 2
```
That would be easier to read and less error prone. But we need to confirm that higher is always better (always maximize)

Reply:
In our case we are only minimizing because our only objectives are ELUC and change. I have updated the function to reflect this by swapping the signs. However, if we want to define domination as a solution doing better in at least one objective and better than or equal to in all the rest we have to do it this way and we can't simply `return False` if we see obj1 == obj2.

Reply:
OK, I've update the docstring to reflect that.